pyproject.toml

```diff
@@ -31,17 +31,21 @@ packaging = "^21.3"
 snowflake = [
     "snowflake-sqlalchemy",
 ]
+postgres = [
+    "sqlalchemy",
+    "psycopg2",
+]
 
 [tool.poetry.group.sql.dependencies]
 great-expectations = { version = "^0.18.4", extras = [
     "databricks",
     "sqlalchemy1",
-    "postgres",
```

Comment:
Removed this because using this extra will install `psycopg2-binary`
https://github.com/great-expectations/great_expectations/blob/develop/reqs/requirements-dev-postgresql.txt